Brokerages/Brokerage.cs

```diff
@@ -54,6 +54,11 @@ public abstract class Brokerage : IBrokerage
         /// </summary>
         public string Name { get; private set; }
 
+        /// <summary>
+        /// Base account currency
+        /// </summary>
+        public string AccountCurrency { get; set; } 
```

Comment:
This doesn't appear to be used in this class. If individual brokerages require this value, then I think it would need to be specified via the individual brokerage factories. This might lead us to require the account currency be part of the algorithm job packet. Setting the account currency in the job packet would also remove the temporal issues during initialization.